dev/com.ibm.ws.jpa.container.v22/bnd.bnd

```diff
@@ -0,0 +1,36 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: JPA 2.2 Container Runtime
+Bundle-SymbolicName: com.ibm.ws.jpa.container.v22
+Bundle-Description: JPA 2.2 Container Runtime; version=${bVersion}
+
+WS-TraceGroup: JPA
+
+Private-Package: \
+  com.ibm.ws.jpa.container.v22.*
+
+-dsannotations: \
+  com.ibm.ws.jpa.container.v22.JPAPCtxtAttributeAccessorV22, \
+  com.ibm.ws.jpa.container.v22.internal.JPA22Runtime
+
+-buildpath: \
+	com.ibm.ws.jpa.container;version=latest,\
+	com.ibm.websphere.javaee.persistence.2.2;version=latest,\
+	com.ibm.wsspi.org.osgi.core;version=latest,\
+	com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
+	com.ibm.tx.core;version=latest,\
+	com.ibm.ws.tx.embeddable;version=latest,\
+	com.ibm.ws.logging.core;version=latest,\
+	com.ibm.ws.container.service;version=latest,\
+	com.ibm.websphere.javaee.transaction.1.1;version=latest
```

Comment:
should depend on transaction.1.2 instead of 1.1

Reply:
Good catch, thanks!